test/rootchain/transactions.js

```diff
@@ -163,6 +163,84 @@ contract('[RootChain] Transactions', async (accounts) => {
         assert.equal(exit[3].toNumber(), 3, "exit's state not set to finalized");
     });
 
+    it("Allows validator to start a fee withdrawal exit", async () => {
+        // non-validators cannot start fee exits
+        let err;
+        [err] = await catchError(rootchain.startFeeExit(txPos[0], {from: accounts[1], value: minExitBond}));
+        if (!err)
+            assert.fail("fee exit start from non-validator");
+
+        // validator cannot start a fee exit without putting a sufficient exit bond
+        [err] = await catchError(rootchain.startFeeExit(txPos[0], {from: authority, value: minExitBond - 100}));
+        if (!err)
+            assert.fail("started fee exit with insufficient bond");
+
+        // cannot start a fee exit for a non-existent block
+        let nonExitentBlockNum = txPos[0] + 100;
+        [err] = await catchError(rootchain.startFeeExit(nonExitentBlockNum, {from: authority, value: minExitBond}));
+        if (!err)
+            assert.fail("started fee exit for non-existent block");
+
+        // validator can start a fee exit with sufficient exit bond
+        let tx = await rootchain.startFeeExit(txPos[0], {from: authority, value: minExitBond});
+
+        let position = 1000000*txPos[0] + 10*(Math.pow(2, 16) - 1);
+        let feeExit = await rootchain.txExits.call(position);
+        assert.equal(feeExit[0].toNumber(), 0, "Incorrect fee exit amount");
+        assert.equal(feeExit[2], authority, "Incorrect fee exit owner");
+        assert.equal(feeExit[3].toNumber(), 1, "Incorrect exit state.");
+
+        // can only start a fee exit for any particular block once
+        [err] = await catchError(rootchain.startFeeExit(txPos[0], {from: authority, value: minExitBond}));
+        if (!err)
+            assert.fail("attempted the same exit while a pending one existed");
+    });
+
+    it("Allows validator to start and finalize a fee withdrawal exit", async () => {
```

Comment:
Can we double check that we can challenge a fee withdrawal by proving a spend of the fee position in that block